tests/functional/src/main/resources/maven-module-mapping.properties

```diff
@@ -80,3 +80,5 @@ mule-api=/mule-api/
 mule-extensions-api=/mule-extensions-api/mule-extensions-api/
 mule-extensions-api-persistence=/mule-extensions-api/mule-extensions-api-persistence/
 mule-extensions-api-xml-dsl=/mule-extensions-api/mule-extensions-api-xml-dsl/
+mule-extensions-api-dsql=/mule-extensions-api/mule-extensions-api-dsql/
```

Comment:
Not really, this is extra mapping is required when you have mule, mule-api. extensions-api in the same project. I can move it to other PR if you want... but this file will go way with Guille's PR about the test runner